test/change.test.js

```diff
@@ -82,6 +82,36 @@ describe('Change', function() {
     });
   });
 
+  describe('Change.rectifyModelChanges - promise variant', function() {
+    describe('using an existing untracked model', function() {
+      beforeEach(function(done) {
+        var test = this;
+        Change.rectifyModelChanges(this.modelName, [this.modelId])
+          .then(function(trackedChanges) {
+            done();
+          })
+          .catch(function(err) {
+            return done(err);
+          });
```

Comment:
The more I think about this, the more I am leaning towards using `.catch` everywhere.

Deciding whether it's ok to pass `done` as the second argument of `then` or whether a new `catch` call is needed is tricky and may lead to subtle bugs. Consider this example:

```js
.then(function(modelInstance) {
  return modelInstance.myMethod();
}, errorHandler);
```

One may incorrectly assume that the `then` callback will never throw, but what if `myMethod` is not a function, perhaps because `modelInstance` is `null` or `undefined`?

Therefore I am arguing we should be using `.catch(done)` everywhere.